integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/mediators/cache/ESBJAVA4318Testcase.java

```diff
@@ -96,7 +99,7 @@ public void testCacheMediatorWithPIs() throws Exception {
 
         ByteArrayOutputStream baos2 = new ByteArrayOutputStream();
         response2.getEntity().writeTo(baos2);
-        String actualValue2 = baos.toString();
+        String actualValue2 = baos2.toString();
```

Comment:
what does baos2 mean? Shall we have a meaningful name?

Reply:
Fixed in #1167